builder.go

```diff
@@ -77,7 +77,7 @@ func fillNodeRec(db *helpers.DagBuilderHelper, r Restorer, node *helpers.FSNodeO
 	}
 
 	if node.NumChildren() == db.Maxlinks() {
```

Comment:
`Numchildren()` could be less than `Maxlinks()`, leading to redundancy blocks not created for some data children nodes

Reply:
Yep, you are right. When I did this, I was thinking about large files where the last corner would be without redundancy node, cause their sizes are not the same, but now I have other solution for that.

Reply:
Builder is not a part of the PR so I consider this comment resolved